README.md

```diff
@@ -75,4 +84,84 @@ No description provided.
 
 ## Contributing
 
+## Code Formatter
```

Comment:
put this in a separate doc file: CONTRIBUTING.md - most users will not be editing the code

Reply:
Indeed, done!